jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -114,6 +116,43 @@ class EndpointK @JvmOverloads constructor(
         put("endpoint_id", id)
     }
 
+    /**
+     * Measures the jitter introduced by the bridge itself (i.e. jitter calculated between
+     * packets based on the time they were received by the bridge and the time they
+     * are sent).  This jitter value is calculated independently, per packet, by every
+     * individual {@link Endpoint} and their jitter values are averaged together
+     * in this static member.
```

Comment:
Comment is wrong (also in original), because "this" isn't the static member.

Reply:
Yeah this stat was never wired up, apparently.  I'll tweak the comment for now and will probably try and bring this stat back in another PR.